pkgs/applications/radio/uhd/python-api.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/host/python/CMakeLists.txt b/host/python/CMakeLists.txt
+index fdf612747..a67631bc0 100644
+--- a/host/python/CMakeLists.txt
++++ b/host/python/CMakeLists.txt
+@@ -174,6 +174,7 @@ add_custom_command(OUTPUT ${TIMESTAMP_FILE}
+   DEPENDS ${PYUHD_FILES})
+ 
+ add_custom_target(pyuhd_library ALL DEPENDS ${TIMESTAMP_FILE} pyuhd)
++set(HAVE_PYTHON_VIRTUALENV FALSE)
```

Comment:
Perhaps instead of this, and the `string(REGEX REPLACE`, we can modify the command upstream uses here:

https://github.com/EttusResearch/uhd/blob/50fa3baa2e11ea3b30d5a7e397558e9ae76d8b00/host/python/CMakeLists.txt#L179-L180

? What do you think? Perhaps upstream will also accept such a change..

Reply:
Do you have a suggestion on how to utilize upstreams direct call to setuptools for this package here? I am rather fresh to nix and still learning about the details.